nms/fbc_js_core/ui/components/ProfileButton.js

```diff
@@ -126,7 +128,7 @@ const ProfileButton = (props: Props) => {
               onClick={() => {
                 GeneralLogger.info(Events.SETTINGS_CLICKED);
                 props.setMenuOpen(false);
-                navigate('settings');
+                navigate(networkId === null ? '/settings' : 'settings');
```

Comment:
This is needed for the special case that a user lands and `/nms` without any networks and wants to go to the settings.